components/typography/styles.js

```diff
@@ -129,10 +132,7 @@ export const _generateBodySmallStyles = (selector) => {
 			max-height: 5rem;
 		}
 		@media (max-width: 615px) {
-			${selector} {
-				font-size: 0.6rem;
-				line-height: 0.9rem;
-			}
+			${unsafeCSS(!inputStyleTweaksEnabled ? `${selector} { font-size: 0.6rem; line-height: 0.9rem; }` : '')}
```

Comment:
Design decided that this responsive rule to reduce font size isn't needed anymore.

Reply:
A slightly unexpected side effect of this change is that the day names in the mini calendar in the vdiffs are the larger size because we just happened to be taking the screenshot at a small screen width.

Reply:
And some help text and tooltips.